iz_helpers/video.py

```diff
@@ -13,9 +19,19 @@ def write_video(file_path, frames, fps, reversed=True, start_frame_dupe_amount=1
     if reversed == True:
         frames = frames[::-1]
 
-    # Get dimensions of the first frames, all subsequent has to be same sized
```

Comment:
please keep the assert, even more, should output frame[0].size and current invalid frame.size, to get a hint about is it a roundness error +-1 px or do we mess up completeley